mentorship ios/ViewModels/ProfileViewModel.swift

```diff
@@ -25,7 +25,7 @@ class ProfileViewModel: ObservableObject {
         needMentoring: false,
         availableToMentor: false
     )
-    @Published var updateProfileResponseData = ProfileModel.UpdateProfileResponseData(message: "")
+    @Published var updateProfileResponseData = ProfileModel.UpdateProfileResponseData(success: false, message: "")
```

Comment:
is this model holding all the data in a way it's directly presented without manipulating them?

Reply:
yes

Reply:
yes